Mobile/starter_project_matador/lib/features/profile page/presentation/widget/posts_widget.dart

```diff
@@ -0,0 +1,214 @@
+import 'package:flutter/material.dart';
+
+//The first Card in My Posts section
+class Posts extends StatefulWidget {
+  const Posts({super.key});
+
+  @override
+  State<Posts> createState() => _PostsState();
+}
+
+class _PostsState extends State<Posts> {
+  @override
+  Widget build(BuildContext context) {
+    var screenWidth = MediaQuery.of(context);
+    return Container(
+      height: screenWidth.size.height * 0.25,
+      child: Card(
+        shape: RoundedRectangleBorder(
+          borderRadius: BorderRadius.circular(25.0),
+        ),
+        child: Column(
+          children: [
+            Padding(
+              padding: const EdgeInsets.fromLTRB(30, 15, 0, 0),
+              child: Row(
+                children: [
+                  const Text(
+                    'My Posts',
+                    style: TextStyle(fontSize: 18, fontWeight: FontWeight.w600),
+                  ),
+                  SizedBox(width: screenWidth.size.width * 0.55),
+                  const Icon(
+                    Icons.grid_view,
+                    color: Color.fromARGB(255, 6, 90, 159),
+                  ),
+                  SizedBox(width: 30),
+                  const Icon(
+                    Icons.toc,
+                    color: Colors.grey,
+                    size: 30,
+                  ),
+                ],
+              ),
+            ),
+            Row(
+              children: [
+                Padding(
+                  padding: const EdgeInsets.fromLTRB(30, 10, 10, 0),
+                  child: Container(
+                    width: 100,
```

Comment:
Avoid using fixed sizing. Make sure you replace it with media query in all the files.